core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
`[{conf,[federate]}]` make bindings to `*.*.*.*`
If we replace `[{conf,[federate]}]` to `[{conf,[federate, {'type', <<"database">>}]}]` this will create bindings to `*.*.database.*` (only database! it not catch any other notifications).